tests/test_redirect.py

```diff
@@ -282,46 +280,44 @@ def test_security_redirect(self, mozwebqa):
         """
         url = mozwebqa.base_url + '/firefox/security'
         response = requests.get(url)
-        Assert.contains('/firefox/desktop/trust/', response.url)
-        Assert.equal(200, response.status_code)
+        assert '/firefox/desktop/trust/' in response.url
+        assert requests.codes.ok == response.status_code
 
     @nondestructive
     @parametrize("locale", ['son', 'zh-CN', 'ta'])
     def test_firefox_new_redirect(self, mozwebqa, locale):
         url = mozwebqa.base_url + '/firefox/new'
         response = requests.get(url, headers={'Accept-Language': locale})
-        Assert.contains(locale, response.url)
+        assert locale in response.url
 
     @nondestructive
     def test_policy_archive_redirect(self, mozwebqa):
         url = mozwebqa.base_url + '/privacy/archive/'
         response = requests.get(url)
-        Assert.equal(response.status_code, 200)
-        Assert.equal(response.history[0].status_code, 301)
-        Assert.true(response.history[0].headers['location'].endswith('/en-US/privacy/archive/'))
+        assert requests.codes.ok == response.status_code
+        assert requests.codes.moved_permanently == response.history[0].status_code
+        assert response.history[0].headers['location'].endswith('/en-US/privacy/archive/')
 
     @nondestructive
     def test_dnt_redirect(self, mozwebqa):
         url = mozwebqa.base_url + '/dnt'
         response = requests.get(url)
-        Assert.equal(response.status_code, 200)
-        Assert.equal(response.history[0].status_code, 301)
-        Assert.true(response.history[-1].headers['location'].endswith('/firefox/dnt/'))
+        assert requests.codes.ok == response.status_code
+        assert requests.codes.moved_permanently == response.history[-1].status_code
```

Comment:
Note that on the next line the old code used `-1` so I was just making it consistent. This is likely a single redirect where 0 and -1 are probably equal. I'm likely going to do some disruptive changes here soon anyway.